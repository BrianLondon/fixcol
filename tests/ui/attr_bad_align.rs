use fixcol_derive::ReadFixed;

#[derive(ReadFixed)]
struct Thing {
    #[fixcol(width = 5, align = "backwards")]
    field: String,
}

pub fn main() {}
