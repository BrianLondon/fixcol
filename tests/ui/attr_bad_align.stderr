error: fixcol-derive error: Expected values for align are "left", "right", or "full".
 --> tests/ui/attr_bad_align.rs:5:33
  |
5 |     #[fixcol(width = 5, align = "backwards")]
  |                                 ^^^^^^^^^^^
